systemtest/containers.go

```diff
@@ -556,6 +556,13 @@ func (c *ElasticAgentContainer) Close() error {
 	return c.container.Terminate(context.Background())
 }
 
+func (c *ElasticAgentContainer) APMServerlog() (io.ReadCloser, error) {
+	return c.container.CopyFileFromContainer(
+		context.Background(),
+		"/usr/share/elastic-agent/state/data/logs/default/apm-server-20220302-1.ndjson",
```

Comment:
is this file name static? It looks like there's a date in there, `20220302`

Reply:
Good catch! I meant to change it before committing, clearly didn't.